packages/odyssey-react-mui/src/Select.tsx

```diff
@@ -295,40 +410,48 @@ const Select = <
                   <CheckIcon />
                 </ListItemSecondaryAction>
               )}
-          </MenuItem>
+          </MuiMenuItem>
         );
       }),
     [hasMultipleChoices, normalizedOptions, internalSelectedValues]
   );
 
   const renderFieldComponent = useCallback(
     ({ ariaDescribedBy, errorMessageElementId, id, labelElementId }) => (
-      <MuiSelect
-        {...inputValues}
-        aria-describedby={ariaDescribedBy}
-        aria-errormessage={errorMessageElementId}
-        children={children}
-        data-se={testId}
-        displayEmpty={
-          inputValues?.value === "" || inputValues?.defaultValue === ""
-        }
-        id={id}
-        inputProps={{ "data-se": testId }}
-        inputRef={localInputRef}
-        labelId={labelElementId}
-        multiple={hasMultipleChoices}
-        name={nameOverride ?? id}
-        onBlur={onBlur}
-        onChange={onChange}
-        onFocus={onFocus}
-        renderValue={hasMultipleChoices ? renderValue : undefined}
-        translate={translate}
-      />
+      <SelectContainer>
+        <MuiSelect
+          {...inputValues}
+          aria-describedby={ariaDescribedBy}
+          aria-errormessage={errorMessageElementId}
+          children={children}
+          id={id}
+          inputProps={{ "data-se": testId }}
+          inputRef={localInputRef}
+          labelId={labelElementId}
+          multiple={hasMultipleChoices}
+          name={nameOverride ?? id}
+          onBlur={onBlur}
+          onChange={onChange}
+          onFocus={onFocus}
+          renderValue={
+            hasMultipleChoices
+              ? (value) => renderValue({ selected: value })
+              : undefined
+          }
+          translate={translate}
+        />
+        {hasMultipleChoices && value && (
+          <PlaceholderValuesContainer odysseyDesignTokens={odysseyDesignTokens}>
+            {renderValue({ selected: value, isPlaceholder: true })}
```

Comment:
It looks like `PlaceholderValuesContainer` doesn't take render props. In that case, why are we passing a `render` function?

In those cases, it should either be inline or a separate component altogether. This just adds indirection making it harder for someone to skim the code and especially hard to someone reviewing a PR (me).